service/k8sclient/k8sclient.go

```diff
@@ -402,26 +412,22 @@ func (c *K8sClient) CreateXtraDBCluster(ctx context.Context, params *XtraDBParam
 		return fmt.Errorf(clusterWithSameNameExistsErrTemplate, params.Name)
 	}
 
-	var secret common.Secret
-	err = c.kubeCtl.Get(ctx, k8sMetaKindSecret, defaultPXCSecretName, &secret)
-	if err != nil {
-		return errors.Wrap(err, "cannot get default PXC secrets")
-	}
-
 	secretName := fmt.Sprintf(pxcSecretNameTmpl, params.Name)
-	pwd, err := randSeq(passwordLength)
+	rootPassword, err := generatePassword(passwordLength)
 	if err != nil {
 		return errors.Wrap(err, "failed to generate password")
 	}
 
 	// TODO: add a link to ticket to set random password for all other users.
-	data := secret.Data
-	data["root"] = []byte(pwd)
-
-	err = c.CreateSecret(ctx, secretName, data)
-	if err != nil {
-		return errors.Wrap(err, "cannot create secret for PXC")
+	secrets := map[string][]byte{
```

Comment:
can you explain this a bit in comments?
i can see that its mapping the same password to different names, are these different components of an XtraDBCluster that share the same password?

Reply:
That's correct and there is comment above to add a link to ticket to set random password for all other users.

Reply:
also should we call the variable `rootPassword` if it is being mapped to things other than `root`? maybe just call it pwd?

Reply:
> can you explain this a bit in comments?
i can see that its mapping the same password to different names, are these different components of an XtraDBCluster that share the same password?


okay I get it it represents the contents of this file that was deleted `pxc-secrets.yaml ` so maybe a comment saying `// secrets is equivalent to the contents of a pxc-secrets.yaml file used to create a cluster ` something like that

Reply:
yes, we still should call it `rootPassword` to make it clear that we use root password for all secrets. and it should be fixed

Reply:
> okay I get it it represents the contents of this file that was deleted pxc-secrets.yaml so maybe a comment saying // secrets is equivalent to the contents of a pxc-secrets.yaml file used to create a cluster something like that

Will do it.